javascript/est-widget.js

```diff
@@ -27,6 +27,11 @@
     }
 
     function main() {
+        var paging_script_tag = document.createElement('script');
+        paging_script_tag.setAttribute("type", "text/javascript");
+        paging_script_tag.setAttribute("src", "javascript/jquery.paging.min.js");
```

Comment:
The problem here though is that this file needs to exist on the web server so that it can be loaded when our plugin wants it. We'd have to tell our users to make sure the file is present, which we don't want to do. We need to package this inline with the rest of the code (i.e. we need an asset pipeline to pre-process our JS code before deployment, something like [broccoli](http://broccolijs.com/)). We could do this with jQuery as well I guess, meaning we wouldn't have to ask our users to include any of our dependencies.